### ISSUE TITLE
tcpdf library error

### ISSUE THREAD

**Issue by tr-j** [GibbonEdu/core#1595]
**Describe the bug**
I can no longer scan the assets directory on the reports module. It results in a blank screen.

In the logs I get many errors, such as: 
[05-Jan-2022 12:45:09 Asia/Yangon] PHP Deprecated:  Required parameter $currentfont follows optional parameter $setbom in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php on line 1899
[05-Jan-2022 12:45:09 Asia/Yangon] PHP Deprecated:  Required parameter $currentfont follows optional parameter $setbom in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php on line 1919
[05-Jan-2022 12:45:09 Asia/Yangon] PHP Deprecated:  Required parameter $currentfont follows optional parameter $str in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php on line 1936
[05-Jan-2022 12:45:09 Asia/Yangon] PHP Deprecated:  Required parameter $currentfont follows optional parameter $str in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php on line 1952
[05-Jan-2022 12:45:09 Asia/Yangon] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_static.php on line 140
[05-Jan-2022 12:45:15 Asia/Yangon] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_static.php on line 140
[05-Jan-2022 12:45:26 Asia/Yangon] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /home/gibbon/public_html/vendor/tecnickcom/tcpdf/include/tcpdf_static.php on line 140

**Expected behavior**
In order to resolve the error I download the current version from Github  and replaced the tcpdf library. The Github library specifically mentions fixed for PHP 8.

The Gibbon version packaged with my install of 22.0.01 is: 6.0.038 (2013-10-06)
The Github version is: 6.4.4 (2021-12-31)  - PHP 8.1 fixes

**Your System**

 - Gibbon version 22.0.01
 - I'm using Mpdf for the renderer (but I guess it used tcpdf during the asset scanning)

**Comment by yookoala** [comment]
Thanks for reporting. Gibbon after #1434 (merged into v23.0.00) will allow you to manually update the tcpdf version.

For v22.0.00, you need to manually remove these lines from composer.json:
https://github.com/GibbonEdu/core/blob/v22.0.01/composer.json#L67-L72

Then you can run this [composer]()getcomposer.org/ command to upgrade tcpdf to its latest version:
```
composer require tecnickcom/tcpdf:"^6.4"
```

**Comment by SKuipers** [comment]
Thanks @tr-j Gibbon  v23.0.00, releasing January 20th, 2022, contains the updated version of TCPDF.
